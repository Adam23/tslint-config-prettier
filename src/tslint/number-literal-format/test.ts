const ns = [.123, 0xABC, 1.20];
