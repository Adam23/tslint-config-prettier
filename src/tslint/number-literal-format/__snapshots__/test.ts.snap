// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (number-literal-format) /src/tslint/number-literal-format/test.ts[1, 13]: Number literal should begin with '0.' and not just '.'.
ERROR: (number-literal-format) /src/tslint/number-literal-format/test.ts[1, 26]: Number literal should not have a trailing '0'.

======
ERROR: (number-literal-format) /src/tslint/number-literal-format/test.ts[1, 20]: Hexadecimal number literal should be uppercase.

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const ns = [.123, 0xABC, 1.20];

======
const ns = [0.123, 0xabc, 1.2];

>>>>>> after
"
`;
